README.md

```diff
@@ -749,9 +749,22 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 ## Disclosures
 
-| Product | Date | CVE | CVSS | Exploits | Report |
-| --- | --: | :-: | --- | ---- | :-: |
-| [SQLite](https://sqlite.org/) | Jul 2022 | [CVE-2022-35737](https://nvd.nist.gov/vuln/detail/CVE-2022-35737) | 7.5 | [Crash](disclosures/CVE-2022-35737/README.md#snprintf-crashc) <br />[Live lock](disclosures/CVE-2022-35737/README.md#snprintf-livelockc) <br />[Code execution](disclosures/CVE-2022-35737/README.md#snprintf-control-pcc) | [💬](https://blog.trailofbits.com/2022/10/25/sqlite-vulnerability-july-2022-library-api/) |
+|Name|Product|Discoverer|Date|ID|Blog|
+|---|---|---|---|---|---|
+|Numbers turned weapons: DoS in Osmosis’ math library|Osmosis|Sam Alws|2023|❌|[💬](https://blog.trailofbits.com/2023/10/23/numbers-turned-weapons-dos-in-osmosis-math-library/)|
+|Security flaws in an SSO plugin for Caddy|caddy-security|Maciej Domanski, Travis Peters, and David Pokora|2023|❌|[💬](https://blog.trailofbits.com/2023/09/18/security-flaws-in-an-sso-plugin-for-caddy/)|
+|ktor Path Traversal|ktor|Vasco Franco|2023|CVE-2022-48476|❔|
+|Specialized Zero-Knowledge Proof failures|Binance's tss-lib; All forks of tss-lib: Joltify, SwipeChain, and ThorChain; Coinbase's kryptology|Opal Wright|2022|❌|[💬](https://blog.trailofbits.com/2022/11/29/specialized-zero-knowledge-proof-failures/)|
+|Forgery in Amis' Alice library|[Amis' alice](https://github.com/getamis/alice#acknowledgments)|Filipe Casal|2022|❌|❔|
+|The issue with ATS in Apple’s macOS and iOS|macOS and iOS|Will Brattain|2022|CVE-2023-38596|[💬](https://blog.trailofbits.com/2023/10/30/the-issue-with-ats-in-apples-macos-and-ios/)|
```

Comment:
We just updated this in the post earlier this week. This one impacted iOS, iPadOS, tvOS, macOS, and watchOS.

Reply:
adressed